README.md

```diff
@@ -132,10 +126,10 @@ This collection can also imported in the [Postman app](https://www.getpostman.co
 helm ls
 ```
 
-2. Run the command below with the appropriate release name to uninstall the deployment:
+2. Run the command below with the appropriate release name to uninstall the deployment and the ingress controller:
 
 ```bash
-helm delete [release-name]
+helm delete [release-name]  $INGRESSRELEASE
```

Comment:
Should this have both [release-name] and $INGRESSRELEASE, or just $INGRESSRELEASE?

Reply:
We need both as we have two release done .. one for ingress and one for hello world .. So only added it to request .. 
not sure whether i need to specific explicitly specific what that release-name means

Reply:
Ah, OK.  Disregard my comment then.